README.md

```diff
@@ -33,7 +33,7 @@ Options:
   -c, --config <configFile>                  configuration file (JSON, JSONC, JS, YAML, or TOML)
   --configPointer <pointer>                  JSON Pointer to object within configuration file (default: "")
   -d, --dot                                  include files/folders with a dot (for example `.github`)
-  -f, --fix                                  fix basic errors (does not work with STDIN)
+  -f, --fix                                  fix basic errors
```

Comment:
Why does this show as changed?

Reply:
The README file shows the result of --help, and I have synced the CLI option changes here.